src/api-service/__app__/onefuzzlib/proxy.py

```diff
@@ -222,8 +222,8 @@ def save_proxy_config(self) -> None:
             ),
             forwards=forwards,
             region=self.region,
-            instrumentation_key=os.environ.get("APPINSIGHTS_INSTRUMENTATIONKEY"),
-            telemetry_key=os.environ.get("ONEFUZZ_TELEMETRY"),
+            instance_telemetry_key=os.environ.get("APPINSIGHTS_INSTRUMENTATIONKEY"),
+            microsoft_telemetry_key=os.environ.get("ONEFUZZ_TELEMETRY"),
```

Comment:
In the future, we should consider renaming this var to include the string `MICROSOFT`.